app/globals.css

```diff
@@ -136,3 +136,9 @@ html.dark {
   transform: translateX(0px);
   transform: translateY(0px);
 }
+
+.text-gradient {
+  background: linear-gradient(45deg, #6366f1, #1d4ed8);
```

Comment:
What if the organization deploying the leaderboard has a different colour theme? #283

Reply:
okay, I will use the colour theme from the tailwind.config.ts file